packages/nc-gui/components/smartsheet/toolbar/ViewActions.vue

```diff
@@ -230,7 +241,14 @@ useMenuCloseOnEsc(open)
       </template>
     </a-dropdown>
 
-    <LazyDlgQuickImport v-if="quickImportDialog" v-model="quickImportDialog" import-type="csv" :import-data-only="true" />
+    <LazyDlgQuickImport
+      v-for="type in quickImportDialogTypes"
+      :key="type"
+      v-model="quickImportDialogs[type].value"
+      :import-type="type"
+      :base-id="bases[0].id"
```

Comment:
A project can include different bases. You should pass the one that the opened table belongs to.